src/components/configEditor/LogsConfig.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { ConfigSection, ConfigSubSection } from '@grafana/experimental';
+import { Input, Field } from '@grafana/ui';
+import { OtelVersionSelect } from 'components/queryBuilder/OtelVersionSelect';
+import { ColumnHint } from 'types/queryBuilder';
+import { versions as otelVersions } from 'otel';
+import { LabeledInput } from './LabeledInput';
+import { CHLogsConfig } from 'types/config';
+import allLabels from 'labels';
+
+interface LogsConfigProps {
+  logsConfig?: CHLogsConfig;
+  onDefaultDatabaseChange: (v: any) => void;
```

Comment:
let's add more specific signatures for all the components below
```suggestion
  onDefaultDatabaseChange: (v: string) => void;
```